### ISSUE TITLE
Exporting chart as bytes array of PNG, JPEG, SVG, and PDF

### ISSUE THREAD

**Issue by Klaudeta** [vaadin/vaadin-charts-flow#264]
It would be a good feature to be able to export the chart as bytes array of PNG, JPEG, SVG, or PDF on server side in order to have it merged with other data to create a single PDF or Excel or whatever file needed.

**Comment by heruan** [comment]
I strongly agree with the need of this, but I also understand the current graph rendering is done client-side by the web component and I guess it would require the server to run Highcharts on an embedded JS engine.

**Comment by Klaudeta** [comment]
Currently exploring the option of having the svg generated client side, have it send server side and trying to use the batik library to the bytes array out of it. As for now integration of the styles into svg seems to be my issue as batik library is not able to transcode those if styles are added using the style tag within svg content

**Comment by alvarezguille** [comment]
@heruan you're right, but actually that was the case for Vaadin Charts for FW6-8 and the SVGGenerator just used command to render un phantomjs and obtain svg from server rendered chart. 
A similar approach should work now with a more modern headless browser.

**Comment by sebdavid** [comment]
The export server of highcharts can be found there : https://github.com/highcharts/node-export-server

You should pay attention to the terms of licence, which must be accepted to use it, because Highcharts is embedded in the server. If I read it right, it can be used for free in a personal way, or if you are a non-profit company. In all other cases, a licence is required. See https://github.com/highcharts/node-export-server/blob/master/msg/licenseagree.msg

Also, mind the fidelity of the conversion (colors, labels position, etc.).
